text/0085-api-keys.md

```diff
@@ -368,8 +368,67 @@ Only the master key allows managing the API keys.
 ```
 "Authorization: Bearer :masterKey"
 ```
+##### Query Parameters
 
-##### Response
+| Field                    | Type                     | Required |
+|--------------------------|--------------------------|----------|
+| `offset`                 | Integer / `null`         | false    |
+| `limit`                  | Integer / `null`         | false    |
+
+###### `offset`
+
+- Type: Integer
+- Required: False
+- Default: `0`
+
+Sets the starting point in the results, effectively skipping over a given number of API keys.
+
+###### `limit`
+
+- Type: Integer
+- Required: False
+- Default: `20`
+
+Sets the maximum number of documents to be returned by the current request.
+
+##### Response Definition
+
+| Field                    | Type                     | Required |
+|--------------------------|--------------------------|----------|
+| `result`                 | Array[Key]               | true     |
+| `offset`                 | Integer                  | true     |
+| `limit`                  | Integer                  | true     |
+| `total`                  | Integer                  | true     |
+
+###### `results`
+
+- Type: Array[Key]
+- Required: True
+
+An array containing the fetched API keys.
+
+###### `offset`
+
+- Type: Integer
+- Required: True
+
+Gives the `offset` parameter used for the query.
+
+###### `limit`
+
+- Type: Integer
+- Required: True
+
+Gives the `limit` parameter used for the query.
+
+###### `total`
+
+- Type: Integer
+- Required: True
+
+Gives the total number of API keys that can be browsed.
```

Comment:
Is this an actual total or an estimated total? I'm guessing actual, but just to be sure.

Reply:
Actual total. There is no notion of approximation here. 🤓

Reply:
Yes!